packages/replay/src/types.ts

```diff
@@ -344,3 +352,34 @@ export interface ReplayPerformanceEntry {
    */
   data?: Record<string, unknown>;
 }
+
+type RequestBody = null | Blob | BufferSource | FormData | URLSearchParams | string;
+
+export type XhrHint = XhrBreadcrumbHint & {
+  xhr: XMLHttpRequest & SentryWrappedXMLHttpRequest;
+  input?: RequestBody;
+};
+export type FetchHint = FetchBreadcrumbHint & {
+  input: HandlerDataFetch['args'];
+  response: Response;
+};
+
+export type ReplayNetworkRequestData = {
+  startTimestamp: number;
+  endTimestamp: number;
+  url: string;
+  method?: string;
+  statusCode: number;
+  requestBodySize?: number;
+  responseBodySize?: number;
+
+  request?: {
+    size?: number;
+    body?: string;
```

Comment:
IMHO for headers it's fine to use an object, as this will ALWAYS be an object, and it makes even more sense to PII strip them separately (as they aren't really related to each other). And we actually also always get them as a POJO form anyhow.

Reply:
sounds good! 
We'll add headers in a follow up PR, we can still look into using them if we want to/need to.

Reply:
@mydea Let's go with POJO - I think redacting the whole body is a worse experience than individual fields.